specification/compute/resource-manager/Microsoft.Compute/2017-03-30/compute.json

```diff
@@ -194,6 +194,34 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Compute/availabilitySets": {
+      "get": {
+        "tags": [
+          "AvailabilitySets"
+        ],
+        "operationId": "AvailabilitySets_ListAll",
```

Comment:
please, provide examples for this operation.

Reply:
Good point; added an example :)

Reply:
@gatneil thanks, I replaced tabs by spaces. Please, don't use tabs :-) I see that the number of errors is increased from 33 to 34. I will check it on Monday.

Reply:
BTW, In this example https://github.com/gatneil/azure-rest-api-specs/blob/b559e04a1c2608884f10d29c8be90df5fe75b464/specification/compute/resource-manager/Microsoft.Compute/2017-03-30/compute.json#L162, the response schema is also `AvailabilitySetListResult` but the "nextLinkName" is `null`.